manifests/b/buchen/portfolio/0.69.1/buchen.portfolio.locale.de-DE.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: buchen.portfolio
+PackageVersion: 0.69.1
+PackageLocale: de-DE
+Publisher: Andreas Buchen
+PublisherUrl: http://abuchen.name
+PublisherSupportUrl: https://github.com/buchen/portfolio/issues
+PackageName: Portfolio Performance
+PackageUrl: https://github.com/buchen/portfolio
+License: EPL-1.0
+LicenseUrl: https://github.com/buchen/portfolio/blob/master/LICENSE
+ShortDescription: Portfolio Performance berechnet die Gesamtperformance Ihres Anlageportfolios.
+ReleaseNotes: |-
+  - New: Performance improvements for 'Statement of Assets' with many columns
+  - New: CSV export for stacked diagrams in the taxonomy view
+  - New: Introduced a new PDF importer for Bourso Bank
+  - Improvements to the PDF importers for Raisin Bank AG, Quirin, Baader Bank, Trade Republic, ING DiBa, JustTrade, Tradegate, Swissquote, FFB, Arkea Direct Bank, sBroker
+
+
```

```suggestion
ReleaseNotes: |-
  - Verbesserung der Performance von der Vermögensaufstellung mit vielen Spalten
  - Neu: CSV Export für Stapel-Diagramme in den Klassifizierungen
  - Neu: Neuer PDF-Importer für Bourso Bank
  - Verbesserungen an den PDF-Importern für Raisin Bank AG, Quirin, Baader Bank, Trade Republic, ING DiBa, JustTrade, Tradegate, Swissquote, FFB, Arkea Direct Bank, sBroker
```

Since this is a locale file for `de-DE`, ReleaseNotes should also be in that locale. I've taken the localized text from https://github.com/portfolio-performance/portfolio/releases/tag/0.69.1

---

Thanks. I have updated the manifest version to 1.6.0, and added separate German and English locales